src/ck-core/ckrdma.C

```diff
@@ -2064,11 +2064,15 @@ void CkPostBufferInternal(void *destBuffer, size_t destSize, int tag) {
 void CkPostNodeBufferInternal(void *destBuffer, size_t destSize, int tag) {
 
   // check if tag exists in posted req node table
+  CmiLock(CksvAccess(_nodeZCPostReqLock));
```

Comment:
Just curious, what compiler / options pointed this out with LTO?

Reply:
I was building with GCC on x86_64 Linux, but the issue manifested as a crash in one of the ZC tests or examples and not a diagnostic message. The crash happened in the implementation of `std::unordered_map` and traced back to these `find()` calls.